ert_shared/storage/extraction_api.py

```diff
@@ -34,12 +35,15 @@ def _extract_and_dump_observations(rdb_api, blob_api):
         facade.get_observation_key(nr) for nr, _ in enumerate(facade.get_observations())
     ]
 
-    measured_data = MeasuredData(facade, observation_keys)
+    if len(observation_keys) > 0:
+        measured_data = MeasuredData(facade, observation_keys)
 
-    measured_data.remove_inactive_observations()
-    observations = measured_data.data.loc[["OBS", "STD"]]
+        measured_data.remove_inactive_observations()
+        observations = measured_data.data.loc[["OBS", "STD"]]
```

Comment:
what are these two lines prepended with? using tab instead of spaces?

Reply:
That's due to block indentation wrt. the `if` statement above